app/models/like.rb

```diff
@@ -0,0 +1,15 @@
+class Like < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Callback
+  after_save :update_post_likes_counter
+
+  private
+
+  # Method
+  def update_post_likes_counter
+    post.update(likes_counter: post.likes.count + 1)
+  end
```

Comment:
This method has the same issue that the `update_user_posts_counter` has. It can also be fixed by removing the + 1 on line 13: 
```
def update_post_likes_counter
    post.update(likes_counter: post.likes.count)
end
```